parsons/ngpvan/email.py

```diff
@@ -0,0 +1,143 @@
+from parsons.etl.table import Table
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Email(object):
+    def __init__(self, van_connection):
+
+        self.connection = van_connection
+
+    def get_emails(
+        self,
+        orderby=None,
+    ):
+        """
+        Get emails.
+
+        Note that it takes some time for the system to aggregate opens and click-throughs,
+        so data can be delayed up to 15 minutes.
+
+        `Args:`
+            orderby : str
```

Comment:
Since the only real choice to be made is choosing to have things sorted ascending or descending by dateModified, would it make more sense to have `orderby` be something like `ascending=True`, and we then send the appropriate `dateModified%20asc` or `dateModified%20desc` value?

Reply:
I'm open to that. We have tried to keep our parameters to match the API docs params. But it does seems like a reasonable change.

Reply:
I'd suggest doing to simplify the API for the end user.